lib/modules/profile/view/profile_page.dart

```diff
@@ -0,0 +1,48 @@
+// ignore_for_file: prefer_const_literals_to_create_immutables
+
+import 'package:flutter/material.dart';
+import 'package:prepowl/_utils/constants/string_constants.dart';
+import 'package:prepowl/modules/profile/view/user_details_ui/complete_profile_form.dart';
+import 'package:prepowl/modules/welcome/view/welcome_screen_ui/background.dart';
+
+import '../../../_utils/res/dimen.dart';
+
+class CompleteProfileScreen extends StatelessWidget {
+  const CompleteProfileScreen({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Background(
```

Comment:
use of background?

Reply:
Its a common screen for showing Background images Ui